src/quicr_server.cpp

```diff
@@ -69,10 +70,13 @@ QuicRServer::QuicRServer(std::shared_ptr<qtransport::ITransport> transport_in,
 }
 
 std::shared_ptr<qtransport::ITransport>
-QuicRServer::setupTransport([[maybe_unused]] RelayInfo& relayInfo)
+QuicRServer::setupTransport([[maybe_unused]] RelayInfo& relayInfo,
+                            qtransport::TransportConfig cfg)
```

Comment:
Should this be &&? Same question for line 39 as well.

Reply:
Same comment above.